x/staking/client/cli/tx.go

```diff
@@ -108,8 +108,6 @@ where we can get the pubkey using "%s tendermint show-validator"
 	cmd.Flags().String(FlagNodeID, "", "The node's ID")
 	flags.AddTxFlagsToCmd(cmd)
 
-	_ = cmd.MarkFlagRequired(flags.FlagFrom)
```

Comment:
is this still required?

Reply:
No it isn't, as we can have a default specified. The SDK already does not enforce --from anymore from `AddTxFlagsToCmd`, so commands themselves should not do it either.
If no default is specified, `GetFromFields` will return an error.